Fix headers descriptions (#134)
aspose-pdf/Aspose.PDF-Documentation: net/advanced-operations/working-with-images/set-default-font-name/_index.md

type: docs
weight: 90
url: /net/set-default-font-name/

> description: This section describes how to set default font name using Aspose.PDF for .NET library.

Reviewer: ```suggestion
description: This section describes how to set the default font name during the conversion process from PDF to image.
```
